include/effects/strip/fireeffect.h

```diff
@@ -199,16 +199,23 @@ class FireEffect : public LEDStripEffect
 
         for (int i = 0; i < LEDCount; i++)
         {
+            auto sum = 0;
+            for (int j = 0; j < CellsPerLED; j++)
+                sum += heat[i*CellsPerLED + j];
+            auto avg = sum / CellsPerLED;
 
-            CRGB color = GetBlackBodyHeatColor(heat[i*CellsPerLED]/(float)std::numeric_limits<uint8_t>::max());
+            #if LANTERN
+                CRGB color = CRGB(avg, avg * .45, avg * .08);
+            #else
+                CRGB color = GetBlackBodyHeatColor(avg/(float)std::numeric_limits<uint8_t>::max());

```

Comment:
What a fancy way to write 255.  I think I'll take the rest of the day off!

Reply:
Haha, yeah, unleashing `std::numeric_limits` on `uint8_t` is like using a motorized mower on a single blade of grass. :)